docs/_bin/generate-notice-binary.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+import yaml
+
+
+outfile = sys.stdout
```

Comment:
Typically, constants are in UPPER_SNAKE_CASE: https://www.python.org/dev/peps/pep-0008/#constants

When is the value `sys.stdout` used? `out_path` is a required positional command-line arg, so this variable should always get initialized to that value. If `sys.stdout` is not used there, then perhaps `print_error` should be printing to stdout instead?